Banking/src/main/java/com/example/Banking/service/TransactionService.java

```diff
@@ -0,0 +1,37 @@
+package com.example.Banking.service;
+
+import com.example.Banking.model.Account;
+import com.example.Banking.model.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+@org.springframework.stereotype.Service
+public class TransactionService {
+
+    @Autowired
+    private TransactionDao transactionDao;
+    @Autowired
+    private AccountDao accountDao;
```

Comment:
Interact with `AccountService` instead of Dao. As new business logics can be added on account updation on service which should also be applicable here.